spec/hydra/works/models/generic_work_spec.rb

```diff
@@ -13,15 +13,22 @@
 
   describe '#generic_works=' do
     it 'should aggregate generic_works' do
-      generic_work1.generic_works = [generic_work2, generic_work3]
-      expect(generic_work1.generic_works).to eq [generic_work2, generic_work3]
+      generic_work1.child_generic_works = [generic_work2, generic_work3]
+      expect(generic_work1.child_generic_works).to eq [generic_work2, generic_work3]
     end
   end
 
-  describe '#generic_files=' do
+  describe '#child_generic_files=' do
```

Comment:
Yeah, I'm just unclear why Hydra-Works should have/new parent_generic_file.  I think GenericFile should always be a leaf node.